### ISSUE TITLE
Date picker doesn't show up with latest version of notion-like-tables

### ISSUE THREAD

**Issue by l3456** [trey-wallis/obsidian-notion-like-tables#580]
### Describe the bug

<img width="188" alt="image" src="https://github.com/trey-wallis/obsidian-notion-like-tables/assets/95742432/89e4be11-69df-477c-8981-3143eee8780d">

Expected: When I hit enter, the date picker should show up and I can select a date. (This was working earlier)

Now: It shows above menu, not sure why it shows format?

### Steps to reproduce

- Create a table
- Add a date column
- Create a new row
- Try to enter data in date column
- Date picker doesn't show up

### Expected behavior

Explained above

### Are you using the mobile app?

No

### Obsidian debug info

SYSTEM INFO:
	Obsidian version: v1.3.5
	Installer version: v1.3.3
	Operating system: Darwin Kernel Version 22.4.0: Mon Mar  6 21:00:17 PST 2023; root:xnu-8796.101.5~3/RELEASE_X86_64 22.4.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Obsidianotion v1.1.25
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 26
	Plugins enabled: 25
		1: Dataview v0.5.56
		2: Kanban v1.5.3
		3: Templater v1.16.0
		4: Auto Link Title v1.4.1
		5: Excalidraw v1.9.3
		6: Editor Syntax Highlight v0.1.3
		7: HTML Reader v1.0.10
		8: Minimal Theme Settings v6.3.1
		9: Hider v1.2.4
		10: Style Settings v1.0.6
		11: Short links v1.1.2
		12: Omnivore v1.5.2
		13: Home tab v1.1.2
		14: Annotator v0.2.10
		15: Obsidian42 - Strange New Worlds (SNW) v1.2.2
		16: File Tree Alternative Plugin v2.3.4
		17: Tasks v4.0.0
		18: QuickAdd v1.2.0
		19: Notion-Like Tables v6.19.0
		20: Fantasy Calendar v3.0.0
		21: Chronology v1.1.4
		22: Release Timeline v1.0.7
		23: Natural Language Dates v0.6.1
		24: Projects v1.16.0
		25: Another Quick Switcher v8.14.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

### Relevant log output

```shell
NA
```

**Comment by trey-wallis** [comment]
Hi @l3456. I removed the date picker because it wasn't very good. I plan on designing my own custom one, but there have been other tasks with higher priority. A simple solution right now would be to just add the default HTML date picker. I will squeeze that in.

**Comment by l3456** [comment]
Yes, please add it back, I really miss this feature and it's quite tedious to enter date manually.
